packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2443,7 +2433,7 @@ export class ContainerRuntime
 				// If we have processed a local op, this means that the container is
 				// making progress and we can reset the counter for how many times
 				// we have consecutively replayed the pending states
-				this.resetReconnectCount(message);
+				this.resetReconnectCount();
```

Comment:
resetting the reconnect count protects against a bug that's really hard to come out of, if the client is not using grouped batching (file size increase due to garbage chuncked ops which fool the container into thinking it's making progress, so the container will go into an endless loop of creating chunks until the client closes the browser). With that in mind, even though `validateAndProcessRuntimeMessage` protects against it, I would either leave another assert here, or inside `resetReconnectCount` to ensure that whatever refactoring happens in the future, we will never ever consume a chuncked op at this point.

Reply:
I've added assert at the top with comments.